src/utils/text.js

```diff
@@ -11,4 +11,19 @@ export function stringifyTurns(turns) {
         }
     }
     return res.trim();
+}
+
+export function toSinglePrompt(turns, system=null, stop_seq='***', model_nickname='assistant') {
+    let messages = turns;
+    if (system) messages.unshift({role: 'system', content: system});
+    let prompt = "";
```

Comment:
You could initialize the prompt with the system message. There's no reason to start the prompt with `system: `.